lidder_test.go

```diff
@@ -23,12 +24,14 @@ type Zuite struct {
 	suite.Suite
 }
 
-func (s *Zuite) TestParseConfiguration() {
+func configFile() (*defs, error) {
```

Comment:
Since you now use this in two tests, it's a great moment to switch to using `SetupTest` instead https://godoc.org/github.com/stretchr/testify/suite#SetupTestSuite, like

```
func (s *Zuite) SetupTest() {
  conf := ...
  defs, err:= parse(...
  if err!=nil {
    panic(err)
  }
  s.defs = defs
}